core/utils/codel.h

```diff
@@ -135,7 +136,12 @@ class Codel final: public Queue<T> {
     return i;
   }
 
-  size_t Capacity() override { return queue_.max_size(); }
+  size_t Capacity() override { 
```

Comment:
Why is Codel's capacity different from the underlying queue capacity. We have already allocated memory (based on the Resize method below). It would be good to at least add a comment explaining this choice, since the current code is surprising.